### PR TITLE
add second parametor for options

### PR DESCRIPTION
**Pull request by tawago**
An image loaded from a different domain need `useCORS: true` option to correctly render the image in the canvas/screenshot image

### Example:

```ts
const Component: FC<Props> = ({}) => {
  const ref = useRef<HTMLDivElement | null>(null);
  const [image, takeScreenShot] = useScreenshot({ type: 'image/jpeg', quality: 1.0 });
  const download = (image: string | void, { extension = 'jpg' } = {}) => {
    if (!image) return;
    const a = document.createElement('a');
    a.href = image;
    a.download = createFileName(extension, `screenshot`);
    a.click();
  };
  const downloadScreenshot = (ref: any) => {
    takeScreenShot(ref.current, {
      allowTaint: false,
      useCORS: true,
    }).then(download);
  };
  
  ...
  
  return (
    <Screen ref={ref}>
      <img src="..."/>
    </Screen>
    <button onClick={() => downloadScreenshot(ref)}>download screenshot</button>
  );
}
```

### CODE REVIEW

**Comment by tawago** [comment]
@vre2h can you take a look at this PR? it's fairly simple and would solve some issues people are reporting.

File: src/index.js [hunk 1]
```diff
@@ -20,12 +20,13 @@ const useScreenshot = ({ type, quality } = {}) => {
   /**
    * convert html node to image
    * @param {HTMLElement} node
+   * @param {Options} html2canvas options
    */
-  const takeScreenShot = (node) => {
+  const takeScreenShot = (node, options = {}) => {
```
Comment (anthonykrivonos) [inline src/index.js:25]: Great idea, this is gonna fix a lot of peoples' problems.

nit: no need for ` = {}`

Comment (tawago) [reply src/index.js:25]: @anthonykrivonos backward compatibility?

Comment (MustafaTRKYLMZ) [reply src/index.js:25]: I think, there is a bug here. When you take screenshot, it has not image. Ony ıt has text and bacground.

Comment (tawago) [reply src/index.js:25]: @MustafaTRKYLMZ
did you give the option like `useCORS: true` as described in the example above??

**Review by PatharrajAnnadurai — APPROVED** [review]

**Review by marcelo-earth — APPROVED** [review]

**Comment by tomimarkus991** [comment]
When will this get merged?

**Comment by tawago** [comment]
@vre2h Could you have a look please?

**Review by Speedwares — APPROVED** [review]

**Comment by martin-dimi** [comment]
Can we please merge this 🙏

**Comment by DawoodFayyaz23** [comment]
please approve this

**Comment by alexandrumogic** [comment]
@vre2h could you merge this?

**Review by ummahusla — APPROVED** [review]

**Comment by vre2h** [comment]
Hi guys,

Sorry for ignoring y'all. I had some tough time with burnout and haven't checked github. Just got an email from @ummahusla. I'll add some of you as maintainers.

**Pull request merged**